client/src/components/Login.jsx

```diff
@@ -55,9 +55,12 @@ const Login = props => {
     setLoading(true);
     doOpenAuth();
   };
+  const closeModal = () => {
+    setLoading(false);
+  };
```

Comment:
I think `setLoading` only deals with the loading component itself, right? We could probably just remove this functionality entirely (from Redux as well) if we're going to remove the loading screen.

Reply:
All references `setLoading`  removed from the app. 

![](https://media.giphy.com/media/j79IZzkdnUwAhWKcys/giphy.gif)